pcsx2/SourceLog.cpp

```diff
@@ -119,15 +119,32 @@ TLD_deci2 = {
 TLD_sysoutConsole = {
 	L"SYSout",		L"System Out",
 	pxDt("Shows strings printed to the system output stream.")
-};
+}
+
+#ifndef DISABLE_RECORDING
+,TLD_recordingConsole = {
+	L"Recording",			L"Recording Console",
+	pxDt("Shows recording related logs and information")
+},
+
+TLD_controlInfo = {
+	L"Controller Info",			L"Controller Info",
+	pxDt("Shows detailed controller input values for port 1, every frame")
+}
+#endif
+; // End init of TraceLogDescriptors
 
 SysConsoleLogPack::SysConsoleLogPack()
-	: ELF		   (&TLD_ELF, Color_Gray)
-	, eeRecPerf	   (&TLD_eeRecPerf, Color_Gray)
+	: ELF				(&TLD_ELF, Color_Gray)
+	, eeRecPerf			(&TLD_eeRecPerf, Color_Gray)
 	, sysoutConsole(&TLD_sysoutConsole, Color_Gray)
-	, eeConsole	   (&TLD_eeConsole)
-	, iopConsole   (&TLD_iopConsole)
-	, deci2		   (&TLD_deci2)
+	, eeConsole			(&TLD_eeConsole)
+	, iopConsole		(&TLD_iopConsole)
+	, deci2				(&TLD_deci2)
```

Comment:
Formatting not needed.

Reply:
Hmm, ok I was just trying to maintain the alignment that was already present in this file https://github.com/PCSX2/pcsx2/blob/master/pcsx2/SourceLog.cpp#L125 It seems as if extra whitespace was already being used to align these statements.

I'll do a pass soon to hopefully address the inconsistent formatting throughout as well as resolve the CMake errors that are relevant to these additions.